selectrum.el

```diff
@@ -1907,19 +1907,21 @@ plus CANDIDATE."
   (let* ((result (cond ((and selectrum--is-crm-session
                              (string-match crm-separator
                                            selectrum--previous-input-string))
-                        (let ((crm
-                               (if (and selectrum--current-candidate-index
-                                        (< selectrum--current-candidate-index
-                                           0))
-                                   candidate
-                                 (with-temp-buffer
-                                   (insert selectrum--previous-input-string)
-                                   (goto-char (point-min))
-                                   (while (re-search-forward
-                                           crm-separator nil t))
-                                   (delete-region (point) (point-max))
-                                   (insert (selectrum--get-full candidate))
-                                   (buffer-string)))))
+                        (let* ((previous-input-string
+                                selectrum--previous-input-string)
+                               (crm
+                                (if (and selectrum--current-candidate-index
+                                         (< selectrum--current-candidate-index
+                                            0))
+                                    candidate
+                                  (with-temp-buffer
+                                    (insert previous-input-string)
+                                    (goto-char (point-min))
+                                    (while (re-search-forward
+                                            crm-separator nil t))
+                                    (delete-region (point) (point-max))
+                                    (insert (selectrum--get-full candidate))
+                                    (buffer-string)))))
```

Comment:
Can you also move `crm-separator` and `(selectrum--get-full candidate)` out of the temp buffer context?

Reply:
I've made some changes in cd67594. Does that look right to you, and would it be better to force push next time?

Reply:
Thank you, looks good to me. You can do it however you think it is appropriate, I typically squash the commits when merging. Can you also add a changelog?

Reply:
Changelog added. Took me a few tries to get the formatting consistent with the rest of the log. Hope it makes sense.

Let me know if anything else is missing :)